migrate-reporting/src/test/java/org/opentestsystem/rdw/ingest/migrate/reporting/step/UpsertExamsIT.java

```
@@ -131,33 +131,4 @@ public void itShouldNoteDelete() {
-            @Sql(executionPhase = AFTER_TEST_METHOD, scripts = {"classpath:MigrateEntitiesFromStagingTearDown.sql"}),
-            @Sql(executionPhase = AFTER_TEST_METHOD, scripts = {"classpath:MigrateCodesFromStagingTearDown.sql"})
-    })
-
-    /**
-     * The exam's completed date is defined as not null in the schema.
-     * If `explicit_defaults_for_timestamp` mySQL setting is turned off, this causes an undesirable behaviour.
-     * This test verifies that this setting is turned on.
-     */
-    @Test(expected = DataIntegrityViolationException.class)
-    public void testExplicitDefaultsForTimestampIsOn() {
-        jdbcTemplate.execute("INSERT INTO reporting_test.exam ( id, type_id, school_year, asmt_id, asmt_version, opportunity, completeness_id,\n" +
-                "                                administration_condition_id, session_id, performance_level, scale_score, scale_score_std_err,\n" +
-                "                                completed_at, import_id, grade_id, student_id, school_id, iep, lep,\n" +
-                "                                section504, economic_disadvantage, migrant_status, eng_prof_lvl, t3_program_type, language_code,\n" +
-                "                                prim_disability_type,\n" +
-                "                                claim1_scale_score, claim1_scale_score_std_err,claim1_category,\n" +
-                "                                claim2_scale_score, claim2_scale_score_std_err,claim2_category,\n" +
-                "                                claim3_scale_score, claim3_scale_score_std_err,claim3_category,\n" +
-                "                                claim4_scale_score, claim4_scale_score_std_err,claim4_category ) VALUES\n" +
-                "(-100, 1,  2016, -99,  null, 1, 1, 1, 'test', 1, 2145, 0.17, null, -88, -98, -89, -1, 1, 1, 0, 0, 1, 'test', 'test', 'eng', null,\n" +
-                "    2000, 0.11, 1, 2100, 0.12, 2, 2500, 0.13, 3, 3000, .15, 4);"
```

The mind boggling thing is that this query behaved as expected on Aurora. But since I cannot explain why, I am afraid to leave it in. 

I have played with different ```@@sql_mode```, made sure that my local and Aurora setting is the same, and behaviour is still different: Aurora does not complain when NULL value is inserted and silently replaces it with the current timestamp.